arangod/Cluster/ClusterComm.h

```diff
@@ -725,15 +731,17 @@ class ClusterComm {
   /// @brief our background communications thread
   //////////////////////////////////////////////////////////////////////////////
 
-  ClusterCommThread* _backgroundThread;
+  std::atomic_uint _roundRobin;
+  std::vector<ClusterCommThread*> _backgroundThreads;
+
+  std::shared_ptr<communicator::Communicator> communicator();
```

Comment:
Do we really need this here, given that the method is implemented in lines 767-769 again?

Reply:
Yes.  This declaration is for the ClusterComm object version.  Line 767 is for the ClusterCommThread object.